src/main/java/seedu/address/model/house/House.java

```diff
@@ -1,105 +1,34 @@
 package seedu.address.model.house;
 
-import seedu.address.commons.util.ToStringBuilder;
-
 /**
  * Represents a House.
  * Guarantees: details are present and not null, field values are validated, immutable.
```

Comment:
This might be irrelevant now?

Reply:
Needed for UnitNumber, etc that are methods of the interface